src/commands/utility/render.js

```diff
@@ -0,0 +1,52 @@
+const { Command, CommandError, SwitchbladeEmbed } = require('../../')
+module.exports = class Render extends Command {
+  constructor (client) {
+    super({
+      name: 'render',
+      aliases: ['show'],
+      category: 'utility',
+      requirements: { guildOnly: true, botPermissions: ['EMBED_LINKS'] },
+      parameters: [{
+        type: 'messageLink', sameGuildOnly: true, forceExists: true, missingError: 'commands:move.missingMessageLink'
+      }]
+    }, client)
+  }
+
+  async run ({ author, t, message }, link) {
+    if (!link.channel.permissionsFor(this.client.user.id).has('VIEW_CHANNEL')) throw new CommandError(t('commands:render.iDontHavePermissionToRead'))
+    if (!link.channel.permissionsFor(author.id).has('VIEW_CHANNEL')) throw new CommandError(t('commands:render.youDontHavePermissionToRead'))
```

Comment:
Can we create boilerplate for this in the messageLink parameter? It's been used in two commands now, and I feel like it could be standarized as `requiredBotChannelPermissions` and `requiredUserChannelPermissions` so we can reuse it

Reply:
I'm working on that...having a lot of problems with it, I'll warn when done.